.github/cloud-samples-tools/test/config_test.go

```diff
@@ -0,0 +1,98 @@
+package test
+
+import (
+	"cloud-samples-tools/pkg/utils"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLoadConfigEmpty(t *testing.T) {
+	path := filepath.Join("testdata", "config", "empty.json")
+	_, err := utils.LoadConfig(path)
+	expectError(t, err)
+}
+
+func TestLoadConfigDefaultValues(t *testing.T) {
+	path := filepath.Join("testdata", "config", "default-values.json")
+	config, err := utils.LoadConfig(path)
+	ok(t, err)
+	expected := utils.Config{
```

Comment:
Same as previous--use table-based or parameterized testing to check multiple different cases.